app/keepers.go

```diff
@@ -367,7 +369,10 @@ func (app *OsmosisApp) InitNormalKeepers(
 
 	// The last arguments can contain custom message handlers, and custom query handlers,
 	// if we want to allow any custom callbacks
-	supportedFeatures := "iterator,staking,stargate"
+	supportedFeatures := "iterator,staking,stargate,osmosis"
```

Comment:
We add the extra feature here, so contract including [osmo-bindings](https://github.com/confio/osmosis-bindings/blob/main/packages/bindings/src/lib.rs#L13-L14) are supported here.